resources/views/ItemEdit/modal/save.blade.php

```diff
@@ -0,0 +1,62 @@
+<style>
+    .card-modal{
```

Comment:
Same as before, It is better to put this to css file.

Reply:
The edit item is Yuya's part, so I will send a message to Yuya.